src/Resource/Shipment.php

```diff
@@ -209,6 +187,46 @@ public function getShipToAddress(): Address
         return $this->shipToAddress;
     }
 
+    /**
+     * @param Address $shipToAddress
+     */
+    public function setShipToAddress(Address $shipToAddress): void
+    {
+        $this->shipToAddress = $shipToAddress;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getDpdNumber(): ?string
+    {
+        return $this->dpdNumber;
+    }
+
+    /**
+     * @param string|null $dpdNumber
+     */
+    public function setDpdNumber(?string $dpdNumber): void
+    {
+        $this->dpdNumber = $dpdNumber;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getDpdDepotCode(): ?string
+    {
+        return $this->dpdDepotCode;
+    }
+
+    /**
+     * @param string|null $dpdDepotCode
+     */
+    public function setDpdDepotCode(?string $dpdDepotCode): void
+    {
+        $this->dpdDepotCode = $dpdDepotCode;
+    }
```

Comment:
zijn deze nog nodig? We nemen ze natuurlijk nu over uit het adresboek item wat op de client zit!? @mwansinck

Reply:
Als er een shipFrom adres opgegeven wordt halen we in de huidige API logica nu deze gegevens niet op omdat het afhaaladres kan afwijken van hun standaard adres waardoor ook de depot informatie anders kan zijn.

Reply:
ja okee, als ze hem zelf opgeven niet idd. Maar als ze de shipfrom leeglaten neemt hij hem wel daar van over toch?

Reply:
Yes